### DISCUSSION TITLE
关于ai回答的语言问题

### DISCUSSION THREAD

**Discussion by qianxifu** [ymcui/Chinese-LLaMA-Alpaca-2#343] [Q&A]
我用text-generation-webui部署了Chinese-Alpaca-2-13B-16K 和Chinese-Alpaca-2-7B两个模型。用中文提问时回答都是英文。怎么设置ai自动使用提问的语言回答？比如使用中文提问就默认使用中文回答。使用英文提问就默认使用英文回答

**Comment by Senthie** [comment]
我也遇到了这个问题 我先训练了Lora 然后将lora 和 alpaca进行模型合并 然后问Llama训练集的问题，Llama的回答是英文掺杂中文
